Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -79,6 +79,8 @@ final class PaymentViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
+    oldOrientation = UIKit.UIDevice.current.orientation
```

Comment:
if the device is in landscape this would return landscape and in line 99 you are setting it to landscape ?. Please can you explain how the `oldOrientation` and `currentOrientation` variables work ?

Reply:
@`viewDidLoad`:  
- `oldOrientation` is capture previous project orientation like `.landscapreLeft` or any type.
- `currentOrientation` for current screen is set to be`.portrait` for device orientation. 

@`viewDidDisappeared`:
- Device orientation is returned to the captured `oldOrientation` which is `.landscapreLeft`.